app/connector/mongodb/src/test/java/io/syndesis/connector/mongo/MongoDBConnectorCountTest.java

```diff
@@ -48,9 +48,9 @@ public void mongoCountSingleTest() {
         collection.insertOne(Document.parse(json2));
         // Given
         String countArguments = "{\"test\":\"single\"}";
-        long count = (long) template().sendBody("direct:start", ExchangePattern.InOut, countArguments);
+        Document result = Document.parse(template.requestBody("direct:start", countArguments, String.class));
         // Then
-        assertEquals(1, count);
+        assertEquals(Integer.valueOf(1), result.getInteger("count"));
```

Comment:
Could we use AssertJ assertions? Something like:
```suggestion
        assertThat(result.getInteger("count")).isEqualTo(1);
```